Automattic/wp-calypso
# Subscription Management: Create "Notify me of new comments" toggle

Closes https://github.com/Automattic/wp-calypso/issues/76203

## Proposed Changes

* Create "Notify me of new comments" toggle
* Create `usePostNotifyMeOfNewCommentsMutation` hook
* Add tracks event for the toggle
* Fix action button position
* Change hover background color to match other menu items
* Fix fill color for icons on hover

<img width="394" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3113712/beb508de-38d6-4c60-b323-0fdc216bcbe9">

## Testing Instructions

* Apply this PR to your local
* Go to http://calypso.localhost:3000/read/subscriptions/comments
* You should see the new toggle on the settings popover
* Toggle on/off and check if the value was updated correctly
* Toggle on and verify if the subscriber receives the comments notification

## Pre-merge Checklist
